pkgs/by-name/cr/crabfit-frontend/package.nix

```
@@ -0,0 +1,109 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchYarnDeps,
+  nodejs,
+  yarn,
+  fixup_yarn_lock,
+  google-fonts,
+  api_url ? "http://127.0.0.1:3000",
+  frontend_url ? "crab.fit",
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "crabfit-frontend";
+  version = "unstable-2023-08-02";
```

```suggestion
  version = "0-unstable-2023-08-02";
```

https://github.com/NixOS/nixpkgs/tree/master/pkgs#package-naming